### PR TITLE
Travis CI.

### PR DESCRIPTION
**Pull request by isht3**
Settings for the flake8 and the CI have been referenced from EvalAI.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -99,3 +99,6 @@ ENV/
```
Comment (RishabhJain2018) [inline .gitignore]: Please remove this file.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (RishabhJain2018) [inline .travis.yml]: @isht3 There are many things missing in this. Can you please take a look at EvalAI's Travis setup for reference.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,33 @@
+attrs==18.1.0
```
Comment (RishabhJain2018) [inline requirements.txt]: I think we don't need all of these requirements. Please add only required ones.

Comment (isht3) [reply requirements.txt]: Sorry, I auto-generated that.

File: .travis.yml (hunk 2 above)
Comment (isht3) [reply .travis.yml]: Yes, I did. I think many of that was for Postgres, Cache, Services, and Coveralls. Which I believe are not required now. Should I add the notification for slack?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+language: python
+
+python:
+  - "3.5"
+
+install:
+  - pip install -r requirements.txt
```
Comment (varunagrawal) [inline .travis.yml]: You might want to add `flake8` to your `requirements.txt` so the below script actually works. :slightly_smiling_face:

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+
+language: python
+
+python:
+  - "3.5"
```
Comment (varunagrawal) [inline .travis.yml]: Add Python 3.6 as well. It would be nice to be forward thinking.

File: .travis.yml (hunk 4 above)
Comment (isht3) [reply .travis.yml]: @varunagrawal Added! :+1:

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  - "3.6"
+
+install:
+  - pip install -r requirements.txt
+
+script:
+  - flake8 ./
+  - py.test
```
[the remark this answers is not in the document]
Comment (isht3) [reply .travis.yml]: Okay! 👍

File: .travis.yml (hunk 2 above)
Comment (RishabhJain2018) [reply .travis.yml]: @isht3 I think we should also add this: https://docs.travis-ci.com/user/reference/trusty/#Container-based-with-sudo%3A-false

Comment (isht3) [reply .travis.yml]: Sorry, I didn't see this comment. I added it

**Comment by isht3** [comment]
@RishabhJain2018 Can you just go through? :D

File: setup.py [hunk 7]
```diff
@@ -32,12 +32,11 @@
```
Comment (RishabhJain2018) [inline setup.py]: @isht3 Is this file a part of this PR?

Comment (isht3) [reply setup.py]: Nope, changed it.

**Pull request merged**